fluent-react/src/void_elements/index.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Void HTML Elements.
+ * Source: https://github.com/pugjs/void-elements
+ *
+ * FIXME We should use this as dependency but for now it doesn't work with our
+ * UMD builds.
+ */
+export default {
+  'area': true,
+  'base': true,
+  'br': true,
+  'col': true,
+  'embed': true,
+  'hr': true,
+  'img': true,
+  'input': true,
+  'link': true,
+  'meta': true,
+  'param': true,
+  'source': true,
+  'track': true,
+  'wbr': true
+};
```

Comment:
also a quick search of void elements produced a larger list than this:
https://github.com/wooorm/html-void-elements/

any reason we're using this smaller list?

Reply:
though this does look suspiciously exactly like what is in the React code: https://github.com/facebook/react/blob/master/packages/react-dom/src/shared/omittedCloseTags.js

with [the addition of `menuitem`](https://github.com/facebook/react/blob/master/packages/react-dom/src/shared/voidElementTags.js)

Reply:
Ah, nice find in React's source. I'll use their file instead of the `void-elements` package because what we really care about here is not a complete list of void elements but rather the list that React uses to throw errors.